app/pods/publications/publication/proofs/requests/controller.js

```diff
@@ -98,6 +98,14 @@ export default class PublicationsPublicationProofsRequestsController extends Con
     proofingActivity.endDate = now;
     const proofingActivitySave = proofingActivity.save();
 
+    if (
+      !this.publicationSubcase.dateReceived ||
+      proofUpload.receivedAtDate < this.publicationSubcase.dateReceived
```

Comment:
Deze conditie moet omgekeerd. De datum op de subcase moet enkel aangepast worden als de nieuwe received-date recenter (dus groter dan) is dan de huidige received-date.